test/event_reporter_test.rb

```diff
@@ -55,10 +48,10 @@ def test_find_SLC_in_CSV
   end
 
   def test_CSV_can_be_written_with_queue
-    skip
     er = EventReporter.new('./data/full_event_attendees.csv')
+    result = er.find_attendees(:zipcode, "90210")
 
-    assert_equal 45, er.write_list
+    assert_equal 1, er.write_list.count
```

Comment:
you could also that the one object in that list has the zipcode of "90210" <---ayeeee I see what you did there.

```assert_equal "90210", er.list.first.zipcode```

Reply:
😄